src/EditorFeatures/Test2/FindReferences/FindReferencesTests.ParameterSymbol.vb

```diff
@@ -534,5 +534,107 @@ end class
 </Workspace>
             Await TestAPIAndFeature(input)
         End Function
+
+        <WorkItem(16757, "https://github.com/dotnet/roslyn/issues/16757")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        <Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.LocalFunctions)>
```

Comment:
@VSadov I am not sure I am following. Could you please elaborate what tests do you have in mind?